protocol.md

```diff
@@ -405,6 +406,10 @@ to expire. If the expiration is known at the creation, the `Upload-Expires`
 header MUST be included in the response to the initial `POST` request.
 Its value MAY change over time.
 
+The Server SHOULD update a files `Upload-Expires` header with each PATCH request
+to prevent file expiration from occuring while the file is still being actively
+uploaded.
```

Comment:
Since the `Upload-Expires` header is not really an entity that you "update", this could also say something like:

> The server SHOULD renew upload's expiration time with each PATCH request, so that the Client doesn't have a time limit for finishing the upload.

Or you could even replace the sentence "Its value MAY change over time" in the previous paragraph with this one.